docs/pass-through.md

```diff
@@ -0,0 +1,237 @@
+# Running Antrea In Pass-Through Mode
+
+Antrea supports chaining with other CNI implementations such as EKS, AKS. In this mode, 
+Antrea enforces k8s network policies, and delegates Pod IP management
+and network connectivity to the primary CNI.
+
+The pass-through mode is designed to be compatible with all major CNI implementations, 
+with emphasis to support cloud provider managed clusters such asAKS, EKS, or potentially
+some flavors of GKE clusters.  
+
+## Deploy and Test Antrea In Cloud 
+Assuming you already have a cloud managed k8s cluster, and have KUBECONFIG environment
+variable set to configuration file of that k8s cluster. You may also refer to [here](cloud.md)
+to create an AKS or EKS cluster on the flight.
+
+Create an Antrea manifest for pass-through mode, for instance
+```bash
+./hack/generate-manifest.sh --encap-mode passThrough > ~/my-yaml/antrea-passthrough.yaml
+```
+
+Adjust defaultMTU, and serviceCIDR values of antrea-agent.conf in the manifest to your cluster, and 
+apply the manifest to the cluster, for instance
+```bash
+kubectl apply -f ~/my-yaml/antrea-passthrough.yaml 
+```
+
+Now Antrea should be plugged into the k8s cluster and enforces any network policies.
+You can now validate Antrea network policy engine by run k8s conformance and network policy tests.
+```bash
+./ci/run-k8s-e2e-tests.sh --e2e-all
+```
+You may need to install a lower version of sonobuoy 
+[here](https://github.com/vmware-tanzu/sonobuoy/releases/), and set environment variable SONOBUOY to
+path to the executable, if you k8s cluster is lower version;
+
+
+### Caveats
+When install Antrea after a k8s cluster already has some Pods running, these pre-installed Pods
+will not have Antrea managing their network policies. If they re-initiate, they then are under Antrea.
+For instance,
+```bash
+kubectl scale deployment coredns --replicas 0 -n kube-system
+kubectl scale deployment coredns --replicas 2 -n kube-system
+```
+
+## Design
+
+### Connecting to CNI Topology
+Antrea is designed to work as network policy plug-in for the following CNI topology models. For
+as long as a CNI implementation fits into one of these two models, Antrea may be inserted for
+network policy in that CNI's environment.
+
+#### Switched CNI Topology
+<img src="/docs/assets/l2-cni.svg" width="600" alt="Antrea Switched CNI">
+
+The above diagram depicts a switched CNI network topology on the left, and what it looks like 
+after Antrea via OVS bridge is inserted into the data path.
+
+In a typical switched CNI topology such as azure AKS, each Pod is connected to a bridge via an veth
```

Comment:
Since we plan to discuss with AKS team on the integration, I suggest not to describe the current implementation for AKS before the design is closed.
Actually I am not certain if we should say we generally support "switched CNI" or not. Feel we might not really support all "switched CNI".

If we assume "passThrough" is only for cloud. It might make sense to describe the Node/Pod network for each cloud one by one? Not sure if readers without enough contexts understand our terms like switched or routed topology, passthrough, etc.

Reply:
I had the same thought as Jianjun actually (second paragraph). It may be better to stick to each cloud rather than try to be generic.

Reply:
From open source perspective, my goal to expose Antrea to as many CNI as possible, so it is not Cloud specific. It is just so that cloud is big use case right now.

But I'd like to hear what are the draws backs to make solution generic.

Reply:
I think specific code could be generic, but user documentation and yaml might not be, unless the implementation can be really apply to multiple clouds/platforms without any changes. Let us discuss to understand better.